src/qml/LocatorItem.qml

```diff
@@ -12,6 +12,9 @@ Item {
   property bool searchFieldVisible: searchField.visible
   property alias locatorModelSuperBridge: locator
 
+  signal searchTermChanged(var searchTerm)
```

Comment:
Can you document this signal (and especially mention the connection to `serchTermHandled`?